nextflow/accession.nf

```diff
@@ -35,20 +35,23 @@ params.help = null
 if (params.help) exit 0, helpMessage()
 
 // Test input files
-if (!params.valid_vcfs || !params.project_accession || !params.instance_id || !params.accession_job_props || !params.public_ftp_dir || !params.accessions_dir || !params.public_dir || !params.logs_dir) {
+if (!params.valid_vcfs || !params.project_accession || !params.instance_id || !params.accession_job_props || !params.public_ftp_dir || !params.accessions_dir || !params.public_dir || !params.logs_dir || !params.accession_job_props.taxonomy_id) {
     if (!params.valid_vcfs) log.warn('Provide validated vcfs using --valid_vcfs')
     if (!params.project_accession) log.warn('Provide a project accession using --project_accession')
     if (!params.instance_id) log.warn('Provide an instance id using --instance_id')
     if (!params.accession_job_props) log.warn('Provide job-specific properties using --accession_job_props')
+    if (!params.accession_job_props.taxonomy_id) log.warn('Provide taxonomy_id in the job-specific properties (--accession_job_props) using field taxonomy_id')
     if (!params.public_ftp_dir) log.warn('Provide public FTP directory using --public_ftp_dir')
     if (!params.accessions_dir) log.warn('Provide accessions directory using --accessions_dir')
     if (!params.public_dir) log.warn('Provide public directory using --public_dir')
     if (!params.logs_dir) log.warn('Provide logs directory using --logs_dir')
     exit 1, helpMessage()
 }
 
-valid_vcfs = Channel.fromPath(params.valid_vcfs)
-
+is_human_study = (params.accession_job_props.taxonomy_id == 9606)
+(valid_vcfs, tabix_vcfs, csi_vcfs) = ( is_human_study
+                     ? [ Channel.empty(), Channel.fromPath(params.valid_vcfs), Channel.fromPath(params.valid_vcfs) ]
+                     : [ Channel.fromPath(params.valid_vcfs), Channel.empty(), Channel.empty() ] )
```

Comment:
That's a bit of a hack which makes it difficult to understand. Can you add a comment explaining what we are doing and how this is working?

Reply:
It's actually an [officially sanctioned Nextflow Pattern](https://nextflow-io.github.io/patterns/index.html#_skip_process_execution), but I agree it's a bit hard to grok and a comment could help.  (assuming we do in fact want to keep this logic in the nextflow, as per the discussion in slack...)

Reply:
added comments